.isort.cfg

```diff
@@ -0,0 +1,5 @@
+[*.py]
+profile=black
+
+#custom
+float_to_top=True
```

Comment:
Do you mean ``force_to_top`` instead??

https://pycqa.github.io/isort/docs/configuration/options/#force-to-top

Reply:
Need also to link ``autoformat`` recipe with .isort.cfg

https://github.com/ITISFoundation/osparc-simcore/blob/b138a1bef1c263cb92a1709fdafe56d2447bd47d/scripts/common.Makefile#L96

Reply:
no force_to_top only moves specific imports to the top. while float_to_top does move any non-indented import to the top: see [https://pycqa.github.io/isort/docs/configuration/options/#float-to-top](https://pycqa.github.io/isort/docs/configuration/options/#float-to-top)